app/src/main/java/nerd/tuxmobil/fahrplan/congress/about/AboutDialog.java

```diff
@@ -15,11 +14,40 @@
 import android.widget.TextView;
 
 import nerd.tuxmobil.fahrplan.congress.BuildConfig;
-import nerd.tuxmobil.fahrplan.congress.MyApp;
 import nerd.tuxmobil.fahrplan.congress.R;
 
 public class AboutDialog extends DialogFragment {
 
+    private static final String BUNDLE_KEY_SCHEDULE_VERSION =
+            BuildConfig.APPLICATION_ID + ".BUNDLE_KEY_SCHEDULE_VERSION";
+    private static final String BUNDLE_KEY_SUBTITLE =
+            BuildConfig.APPLICATION_ID + ".BUNDLE_KEY_SUBTITLE";
+    private static final String BUNDLE_KEY_TITLE =
+            BuildConfig.APPLICATION_ID + ".BUNDLE_KEY_TITLE";
+
+    public static AboutDialog newInstance(
+            @NonNull String scheduleVersion,
+            @NonNull String subtitle,
+            @NonNull String title
+    ) {
+        Bundle arguments = new Bundle();
+        arguments.putString(BUNDLE_KEY_SCHEDULE_VERSION, scheduleVersion);
+        arguments.putString(BUNDLE_KEY_SUBTITLE, subtitle);
+        arguments.putString(BUNDLE_KEY_TITLE, title);
+        AboutDialog dialog = new AboutDialog();
+        dialog.setArguments(arguments);
+        return dialog;
+    }
+
+    @NonNull
+    private String scheduleVersionText = "";
```

Comment:
Why do you use the Text postfix here, but not in the newInstance parameter names? I suggest to remove Text postfix.

Reply:
The `Text` postfix clearly identifies the text variable and helps to distinguish the text from a `TextView` variable when the text is assigned to the view.

Example:

``` kotlin
TextView scheduleVersion = findViewById(R.id.schedule_version);
scheduleVersion.setText(scheduleText);
```

Another way to avoid naming conflicts would be to rename the `TextView` variable.

``` kotlin
TextView scheduleVersionView = findViewById(R.id.schedule);
scheduleVersionView.setText(schedule);
```

Reply:
I see. I was just surprised that two variables representing the same have different naming schemes. Maybe the solution with the View suffix, would not have bothered me (because text variables would have same names).